sources/gdal/large_image_source_gdal/__init__.py

```diff
@@ -942,9 +942,9 @@ def _getRegionBounds(self, metadata, left=None, top=None, right=None,
             if right is None:
                 right = bounds['xmax'] if width is None else left + width
             if top is None:
-                top = bounds['ymax'] if bottom is None or width is None else bottom - width
+                top = bounds['ymax'] if bottom is None or height is None else bottom + height
             if bottom is None:
-                bottom = bounds['ymin'] if width is None else top + width
+                bottom = bounds['ymin'] if height is None else top - height
```

Comment:
- and + where inverted as well. We agree that height is positive ?

Reply:
`width` should be `height` in the `bottom =` line, but bottom > top depends on the coordinate system.  I'll have to investigate further.